packages/elements/test/integration/lib/SseServer.ts

```diff
@@ -0,0 +1,47 @@
+import http, { Server } from 'http';
+
+export class SseTestServer {
+  private server: Server | undefined;
+  private response: http.ServerResponse | undefined;
+
+  public start() {
+    this.server = http.createServer((req, res) => {
+      if (req.url === '/') {
+        this.response = res;
+        res.writeHead(200, {
+          'Content-Type': 'text/event-stream',
+          'Cache-Control': 'no-cache',
+          Connection: 'keep-alive',
+          'Access-Control-Allow-Origin': '*',
+        });
+      }
+    });
+    this.server.listen();
+  }
+
+  public send(event: { name: string; data: object }) {
```

Comment:
Something like this:
```suggestion
  public sendMutationResult(mutation: Mutation) {
```